paddle/fluid/platform/gpu_info.cc

```diff
@@ -31,13 +30,32 @@ constexpr static float fraction_of_gpu_memory_to_use = 0.92f;
 constexpr static float fraction_of_gpu_memory_to_use = 0.5f;
 #endif
 
+constexpr static float fraction_reserve_gpu_memory = 0.05f;
+
 DEFINE_double(fraction_of_gpu_memory_to_use, fraction_of_gpu_memory_to_use,
               "Allocate a trunk of gpu memory that is this fraction of the "
               "total gpu memory size. Future memory usage will be allocated "
               "from the trunk. If the trunk doesn't have enough gpu memory, "
               "additional trunks of the same size will be requested from gpu "
               "until the gpu has no memory left for another trunk.");
 
+DEFINE_uint64(gpu_init_memory_in_mb, 0ul,
```

Comment:
Maybe we can change it to be `initial_gpu_memory_in_mb` because we have already have a flag named `initial_cpu_memory_in_mb`. BTW, change the type to be `double`.